api/http/proxy/transport.go

```diff
@@ -38,6 +40,14 @@ func newSocketTransport(socketPath string) *http.Transport {
 	}
 }
 
+func newNamedPipeTransport(namedPipePath string) *http.Transport {
+	return &http.Transport{
+		Dial: func(proto, addr string) (conn net.Conn, err error) {
+			return npipe.Dial(namedPipePath)
```

Comment:
Issue might come from here... Looks like @StefanScherer is using github.com/Microsoft/go-winio here. You might want to give it a try.

See: https://github.com/portainer/portainer/compare/develop...StefanScherer:npipe_support#diff-d678b9df77b49ad5bb4195768e75b3afR45